chimera_app/functions.py

```diff
@@ -20,11 +20,11 @@ def sanitize(string):
 
 def load_shortcuts(platform):
     shortcuts = []
-    se = BuddyContext()
+    se = ChimeraContext()
     ensure_directory(se.SHORTCUT_DIRS[0])
 
     shortcuts_file = se.SHORTCUT_DIRS[0] + \
-        "/steam-buddy.{platform}.yaml".format(platform=platform)
```

Comment:
Will need backwards compatibility for this as well I think?

Reply:
The yaml files would work in shortcuts no matter what.
But we can make the migration move those files too.